### PR TITLE
german translation

### PR DESCRIPTION
**Pull request by Rahbaran**
@sbe: please review my translation.

### CODE REVIEW

File: server/mfAll.js [hunk 1]
```diff
@@ -1,513 +1,177 @@
...
-      "text": "Change to \"Published\" when course should be viewable to others.",
-      "ctime": 1426503739075,
-      "mtime": 1426503739075,
-      "revisionId": "CAzRihdNHkA672qvY",
-      "file": "./client/templates/teach/teach.html",
-      "line": 32,
-      "template": "teach",
-      "_id": "2z4YiiBwi58M8dRRN"
+      "lang": "fi",
+      "text": "Valitse joitain tägeä (Esimerkikis JavaScript, MAtematiikka)",
```
Comment (brylie) [inline server/mfAll.js:60]: Matematiikka has two capital letters. The 'A' needs to be lowercase.

File: server/mfAll.js [hunk 2]
```diff
@@ -1,513 +1,177 @@
...
-      "ctime": 1426503739075,
-      "mtime": 1426503739075,
-      "revisionId": "d7pSmHWzkofe9Bexd",
-      "file": "./client/templates/teach/teach.html",
-      "line": 27,
-      "template": "teach",
-      "_id": "iwsb4MBomDhN9QimD"
-    },
-    "add new course form published label": {
-      "key": "add new course form published label",
```
Comment (brylie) [inline server/mfAll.js:88]: Keys are still using spaces here. Did this work start from the current development branch? Recent changes to the development branch have fixed these keys.

**Comment by Rahbaran** [comment]
@brylie i'm going to start from scratch anyway. hopefully, all will be ok then.

**Comment by sebbel** [comment]
What's the status on this? Need help Amir @Rahbaran ?

**Comment by Rahbaran** [comment]
@brylie : are you sure that the dev branch doesn't use any space? I still see it somehow:
https://github.com/Crowducate/crowducate-next/edit/development/server/mfAll.js

**Comment by brylie** [comment]
@Rahbaran where do you see spaces in the keys? What line number(s)?

**Comment by brylie** [comment]
In the [development branch Teach.html](https://github.com/Crowducate/crowducate-next/blob/development/client/templates/teach/teach.html), all keys conform to the camel case convention. We need to identify any keys in the template files that have spaces, if any.

**Comment by brylie** [comment]
I asked for clarification about what we should see in the mfAll.js and mfExtract.js files when running mf_extract and adding localizations.

**Pull request closed without merging**